app/TerminalView.m

```diff
@@ -127,7 +128,7 @@ - (void)installTerminalView {
     for (int i = 0; i < sizeof(HANDLERS)/sizeof(HANDLERS[0]); i++) {
         [webView.configuration.userContentController addScriptMessageHandler:handler name:HANDLERS[i]];
     }
-    webView.frame = self.bounds;
+    webView.frame = CGRectMake(0, 0, screen.width + 13, self.bounds.size.height + 12);
```

Comment:
I'm not sure why the reported screen size is smaller than it is. I double checked the safe areas and bounds of the superviews but no idea.